lua/monoglow/groups/blink-cmp.lua

```diff
@@ -0,0 +1,22 @@
+local M = {}
+
+M.url = "https://github.com/Saghen/blink.cmp"
+
+function M.get(c, opts)
```

Comment:
Can you remove unused `opts`? I know I also have it on `cmp.lua` but I have to remove it :) 

```suggestion
function M.get(c)
```